tests/builder/conflicts.rs

```diff
@@ -296,6 +298,26 @@ fn required_group_conflicts_with_arg() {
     }
 }
 
+#[test]
```

Comment:
I'm not sure if this is the best place to put this test.

Reply:
I think it fits to be in this file.  Exact location in the file seems good enough for now.